.travis.yml

```diff
@@ -120,7 +121,8 @@ script:
     fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
     python -m pytest --verbose --capture=no --cov-report term-missing --cov=vidgear vidgear/tests/;
+    flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics;

```

Comment:
@cclauss **CLI Tests failing since flake8 in not in path. Kindly replace `flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics;` with `python -m flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics;`**